internal/cassandra.go

```diff
@@ -41,6 +41,16 @@ type Database interface {
 	RemoveFromTagIndex(tagKey, tagValue string, metricKey api.MetricKey) error
 }
 
+type DatabaseGraphiteStore interface {
```

Comment:
why do we have sudden hierarchy? i feel like we don't really gain much from doing this.